<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 299, in run
    self.check_source(source, "source")
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_url.py", line 68, in check_source
    self._check_url(url, section)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_url.py", line 59, in _check_url
    if url.lower().startswith("http://"):
       ^^^^^^^^^
AttributeError: 'CommentedSeq' object has no attribute 'lower'

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_6dyv8r08ua/clone` failed. (See above for error)
</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,3 +1,6 @@
 aggregate_check: false
 
 upload_channels: [r]
+
+use_prefect_platforms:
+    - linux-64

